addons/maptools/functions/fnc_updateMapToolMarkers.sqf

```diff
@@ -25,15 +25,8 @@ if (GVAR(plottingBoard_Shown) == 0) then {
             _y set [4, 0];
         };
     } forEach GVAR(plottingBoard_markers);
-};
-
-if (((GVAR(mapTool_Shown) == 0) && {GVAR(plottingBoard_Shown) == 0}) || {
-    private _uniqueItems = ACE_player call EFUNC(common,uniqueItems);
-
-    !(("ACE_MapTools" in _uniqueItems) || {"ACE_PlottingBoard" in _uniqueItems})
-}) exitWith {};
-
-if (GVAR(plottingBoard_Shown) > 0) then {
+} else {
+    if (!("ACE_PlottingBoard" in (ACE_player call EFUNC(common,uniqueItems)))) exitWith {};
```

Comment:
```suggestion
    if !("ACE_PlottingBoard" call EFUNC(common,hasItem)) exitWith {};
```

Reply:
https://github.com/acemod/ACE3/blob/master/addons/common/functions/fnc_hasItem.sqf
takes 2 args

Reply:
I can't believe I looked up the function before posting _and still got it wrong_. Updated suggestion.